source/CardGames.py

```diff
@@ -13,6 +12,14 @@ def find_root_dir():
     cwd = os.path.join( cwd, '..')
   return cwd
 
+
+def showWinner(players):
+    arr = {}
+    for player in players:
+      arr[player.calculateHand()] = player
+    return arr[max(arr.keys())].name
+
```

Comment:
Well written. Works as intended. however I am thinking that in the end it might be a better idea to return the player object instead of just the name. Since we might need to do other things with the player in the future.

Reply:
Unless it would be our intent to just print the winner. But considering that in the future the player that wins will have to receive the money from winning it might be better to return just the player object so that we can add money in the future.

Reply:
This looks really clean. I do think Roy is right though, we will probably want it to return the player object, not just the name.